_data/projects.yml

```diff
@@ -35,6 +35,20 @@
   contact: Bryan Harter, Michael Liu, David Stansby, Michael Hirsch
   keywords: ["magnetosphere","ionosphere_thermosphere_mesosphere","specific"]
 
+- name: CCSDSPy
+  code: https://github.com/ccsdspy/ccsdspy/
+  description: A Python package for reading CCSDS (spacecraft) packet data.
+  docs: "https://docs.ccsdspy.org/en/latest/#"
+  logo: https://docs.ccsdspy.org/en/latest/_static/logo.png
+  contact: Daniel da Silva
+  keywords: ["ccsds","specific", "heliosphere", "ionosphere_thermosphere_mesosphere", "magnetosphere"]
```

Comment:
Looks good! "ccsds" wasn't a valid keyword from the taxonomy but I added it so it is now 👍